charts/pathfinder-rpc-server/values-prd.yaml

```
@@ -0,0 +1,15 @@
+# Blockchain Indexer
+blockchainIndexer:
+ indexerWebsocketURL:      http://rpc.circlesubi.id
+ indexerRPCGatewayURL:     https://rpc.circlesubi.id
```

```suggestion
 indexerWebsocketURL:      http://rpc.circlesubi.id
 indexerRPCGatewayURL:     https://rpc.circlesubi.id
```

@jaensen Should these values be public? I don't recall if we ever announced them openly, but here.

---

no no they shoould I removed this file already